shoop/utils/iterables.py

```diff
@@ -29,11 +29,8 @@ def batch(iterable, count):
     """
     Yield batches of `count` items from the given iterable.
 
-    >>> for x in batch([1, 2, 3, 4, 5, 6, 7], 3):
-    >>>   print(x)
-    [1, 2, 3]
-    [4, 5, 6]
-    [7]
+    >>> tuple(x for x in batch([1, 2, 3, 4, 5, 6, 7], 3))
```

Comment:
I don't really know exactly. `flake8 --doctest` yielded an error about it.

Now that I added also `--show-source` to flake8 invocation I got more info:
```
(shoop-py3)tuomas@laatta:~/projects/shoop/project (tox-coverage-and-doctests) 0 $ flake8 --doctest --show-source 
./shoop/utils/iterables.py:32:51: F999 syntax error in doctest
    >>> for x in batch([1, 2, 3, 4, 5, 6, 7], 3):
                                                 
^
./shoop/utils/iterables.py:33:11: F999 syntax error in doctest
    >>>   print(x)
          ^
```